blazor/datepicker/mask-support.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: post
+title:  Mask Support in Blazor DatePicker Component | Syncfusion
+description: Checkout and learn here all about Mask Support in Syncfusion Blazor DatePicker component and much more.
+platform: Blazor
+control: DatePicker
+documentation: ug
+---
+
+# Mask Support in Blazor DatePicker Component
+
+The [EnableMask]() property in the Syncfusion Blazor Datepicker component allows you to enable or disable the masking functionality of the input field. When enabled, the component will display the input field as a masked textbox, with a specific pattern to enter the date value.
+
+By default, the "EnableMask" property is set to `false`.
+
+You can also customize the mask pattern used by the component by using the [Format](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Calendars.SfDatePicker-1.html#Syncfusion_Blazor_Calendars_SfDatePicker_1_Format) property.
+
+The following example demonstrates default format of DatePicker component with mask.
+
+{% highlight Razor %}
+
+{% include_relative code-snippet/DatePickerMask.razor %}
+
+{% endhighlight %}
+
+
+
+![Blazor DatePicker with EnableMask](./images/DatePickerMask.gif)
+
+## MaskPlaceholder
+
+The [DatePickerMaskPlaceholder]() class in the Syncfusion Blazor Datepicker component is used to customize the maskplaceholders displayed in the input field when the masking functionality is enabled. 
```

Comment:
The [DatePickerMaskPlaceholder](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Calendars.DatePickerMaskPlaceholder.html) directive sets the placeholder text for each segment of the date format in a `DatePicker` component. It can be used to provide additional context or instructions to the user about the format that is expected for the input. To use the directive, include it in the component's configuration along with the [EnableMask](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Calendars.SfDatePicker-1.html#Syncfusion_Blazor_Calendars_SfDatePicker_1_EnableMask) property.